f5_openstack_agent/lbaasv2/drivers/bigip/esd_filehandler.py

```diff
@@ -0,0 +1,245 @@
+# coding=utf-8
+# Copyright 2014-2017 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import glob
+import json
+import os
+import types
+
+from oslo_log import log as logging
+
+
+from f5_openstack_agent.lbaasv2.drivers.bigip import exceptions as f5_ex
+from f5_openstack_agent.lbaasv2.drivers.bigip.resource_helper import \
+    BigIPResourceHelper
+from f5_openstack_agent.lbaasv2.drivers.bigip.resource_helper import \
+    ResourceType
+
+LOG = logging.getLogger(__name__)
+
+
+class EsdJSONFileRead(object):
```

Comment:
Maybe it makes sense to merge the EsdJSONFileRead class with the EsdJSONValidation.